fenix/app/src/main/res/values/strings.xml

```diff
@@ -2068,6 +2068,150 @@
+    <!-- Title for error card displayed to the user when the device is disconnected from the network. -->
+    <string name="review_quality_check_no_connection_title" tools:ignore="UnusedResources">No network connection</string>
+    <!-- Text for body of error card displayed to the user when the device is disconnected from the network. -->
+    <string name="review_quality_check_no_connection_body" tools:ignore="UnusedResources">Check your network connection and then try reloading the page.</string>
+    <!-- Title for card displayed to the user for products whose reviews were not analyzed yet. -->
+    <string name="review_quality_check_no_analysis_title" tools:ignore="UnusedResources">No analysis for these reviews yet</string>
+    <!-- Text for the body of card displayed to the user for products whose reviews were not analyzed yet. First parameter is the Fakespot product name. In the phrase "Fakespot by Mozilla", "by" can be localized. Does not need to stay by. The word analyzer when used here is an intentional use of that word. It reflects what this tool is called on Fakespot.com. If possible, a different word should be used for the Fakespot tool (Launch analyzer on Fakespot.com) than checker, which is used in the name of the Firefox feature (Review checker). If that is not possible - if these terms are not well differentiated - that is OK. First parameter is the Fakespot product name. In the phrase "Fakespot by Mozilla", "by" can be localized. Does not need to stay by. -->
+    <string name="review_quality_check_no_analysis_body" tools:ignore="UnusedResources">Launch the %s by Mozilla analyzer and you’ll know in about 60 seconds whether this product’s reviews are reliable.</string>
+    <!-- Clickable text for the body of card displayed to the user for products whose reviews were not analyzed yet. This should link to Fakespot and trigger a product analysis. "Fakespot.com" intentionally hardcoded. The word analyzer when used here is an intentional use of that word. It reflects what this tool is called on Fakespot.com. If possible, a different word should be used for the Fakespot tool (Launch analyzer on Fakespot.com) than checker, which is used in the name of the Firefox feature (Review checker). If that is not possible - if these terms are not well differentiated - that is OK. First parameter is the Fakespot product name. In the phrase "Fakespot by Mozilla", "by" can be localized. Does not need to stay by. -->
+    <string name="review_quality_check_no_analysis_link" tools:ignore="UnusedResources">Launch analyzer on Fakespot.com</string>
+    <!-- Headline for review quality check contextual onboarding card. -->
+    <string name="review_quality_check_contextual_onboarding_title">Try our trusted guide to product reviews</string>
+    <!-- Description for review quality check contextual onboarding card. Parameters are for retailer names (e.g. Amazon, Walmart). -->
```

Comment:
The comment is incomplete, as there's at least one parameter for Fakespot, and there should be one for Firefox.

Reply:
I don't see the word Fakespot being used in this string. Is this referring to another string?

Reply:
I misread the string, I thought it was referring to Fakespot, but it's one of the 3 ecommerce sites.